src/app/shared/reactions/reactions.component.ts

```diff
@@ -57,21 +73,20 @@ export class ReactionsComponent implements OnChanges {
 
   loadReactionsAmount() {
     this.reactionService.getAmountOfTypes(this.resource).subscribe(returnedAmounts => {
-      returnedAmounts.forEach(typeAmount => {
-        if (typeAmount.type === ReactionType.Like) {
-          this.likesAmount = typeAmount.amount;
-        } else {
-          this.dislikesAmount = typeAmount.amount;
-        }
-      });
+      const typeAmounts = returnedAmounts[0];
```

Comment:
`this.reactionService.getAmountOfTypes` this function should return a single value instead of an array so you won't have to extract the first element

Reply:
Done